diff -urNp setup-2.8.16-orig/csh.login setup-2.8.16/csh.login
--- setup-2.8.16-orig/csh.login	2010-03-26 13:10:06.000000000 +0100
+++ setup-2.8.16/csh.login	2010-03-26 16:36:11.000000000 +0100
@@ -2,12 +2,10 @@
 
 # System wide environment and startup programs, for login setup
 
-if ($?PATH) then
-	if ( $uid == 0 ) then
-		setenv PATH "/sbin:/usr/sbin:/usr/local/sbin:/bin:/usr/bin:/usr/local/bin"
-	else
-		setenv PATH "/bin:/usr/bin:/usr/local/bin:/sbin:/usr/sbin:/usr/local/sbin"
-	endif
+if ( $uid == 0 ) then
+	setenv PATH "/sbin:/usr/sbin:/usr/local/sbin:/bin:/usr/bin:/usr/local/bin"
+else
+	setenv PATH "/bin:/usr/bin:/usr/local/bin:/sbin:/usr/sbin:/usr/local/sbin"
 endif
 
 setenv HOSTNAME `/bin/hostname`
