diff -urNp setup-2.7.5-orig/csh.login setup-2.7.5/csh.login
--- setup-2.7.5-orig/csh.login	2008-11-27 13:03:16.000000000 +0100
+++ setup-2.7.5/csh.login	2008-12-10 17:00:21.000000000 +0100
@@ -17,10 +17,6 @@ endif
 setenv HOSTNAME `/bin/hostname`
 set history=1000
 
-if ( ! -f $HOME/.inputrc ) then
-	setenv INPUTRC /etc/inputrc
-endif
-
 if ( -d /etc/profile.d ) then
         set nonomatch
         foreach i ( /etc/profile.d/*.csh )
diff -urNp setup-2.7.5-orig/profile setup-2.7.5/profile
--- setup-2.7.5-orig/profile	2008-11-27 13:03:16.000000000 +0100
+++ setup-2.7.5/profile	2008-12-10 17:00:02.000000000 +0100
@@ -42,11 +42,7 @@ fi
 HOSTNAME=`/bin/hostname 2>/dev/null`
 HISTSIZE=1000
 
-if [ -z "$INPUTRC" -a ! -f "$HOME/.inputrc" ]; then
-    INPUTRC=/etc/inputrc
-fi
-
-export PATH USER LOGNAME MAIL HOSTNAME HISTSIZE INPUTRC
+export PATH USER LOGNAME MAIL HOSTNAME HISTSIZE
 
 for i in /etc/profile.d/*.sh ; do
     if [ -r "$i" ]; then
@@ -58,7 +54,5 @@ for i in /etc/profile.d/*.sh ; do
     fi
 done
 
-export PATH
-
 unset i
 unset pathmunge
